docs/rtk-query/usage/polling.mdx

```diff
@@ -14,6 +14,10 @@ description: 'RTK Query > Usage > Polling: re-fetching data on a timer'
 
 Polling gives you the ability to have a 'real-time' effect by causing a query to run at a specified interval. To enable polling for a query, pass a `pollingInterval` to the `useQuery` hook or action creator with an interval in milliseconds:
 
+Polling additionally has the ability to skip sending requests while the window is out of focus. To enable this behavior, pass `skipPollingIfUnfocused: true` to the `useQuery` hook or action creator.
```

Comment:
would be cool to move this to a tip admonition or something, after the codeblock - along with an example for the sake of it

Reply:
most recent commit should represent this change. I added it to the existing example with some notes but I can move it to its own as well if preferred.